mill

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env sh
+
+# This is a wrapper script, that automatically download mill from GitHub release pages
+# You can give the required mill version with --mill-version parameter
+# If no version is given, it falls back to the value of DEFAULT_MILL_VERSION
+
+DEFAULT_MILL_VERSION=0.2.7-10-db8525
+
+set -e
+
+if [ "x$1" = "x--mill-version" ] ; then
+  shift
+  if [ "x$1" != "x" ] ; then
+    MILL_VERSION="$1"
+    shift
+  else
+    echo "You specified -v without a version."
+    echo "Please provide a version that matches one provided on"
+    echo "https://github.com/lihaoyi/mill/releases"
```

Comment:
Does this only support stable versions, or unstable versions too? It could be handy for our own development to allow a flag to easily pass in an unstable version so we can test our own code that hasn't been tagged with a numbered release

Reply:
It supports all downloadable version, also these n-commits after-release downloads. In fact, I wrote the script exactly for that use case, called it `mill-launcher` and then realized, that it's already the solution for #476.